kivymd/uix/imagelist.py

```diff
@@ -148,7 +148,7 @@ def build(self):
         x: root.x
         y: root.y if root.overlap or root.box_position == 'header' else box.top
 
-    BoxLayout:
+    MDBoxLayout:
```

Comment:
Using MDBoxLayout instead of BoxLayout needs to be tested.

Reply:
I tested this using all the examples from the documentation